community/index/src/main/java/org/neo4j/index/gbptree/GenSafePointer.java

```diff
@@ -47,6 +47,8 @@
     // unsigned int
```

Comment:
I don't quite understand. It's not just about this class, GSP is a core concept/term so it's a much bigger rename if that is to be done. We have an action point for taking a naming approach around gen vs generation and the likes later on.